src/resources/README.md

```diff
@@ -91,3 +91,16 @@ ffmpeg -display_vflip -i temp.mp4 -c copy four-colors-vp9-bt601-vflip.mp4
 rm temp.mp4
 
 ```
+
+The test jpg files were generated with by exiftool cmds below:
+```
+// Generate jpg picture with 90 cw rotation metadata
+exiftool -Orientation#=6 four-colors.jpg -o .\four-colors-rotate-90-cw.jpg
+
+// Generate jpg picture with 180 cw rotation metadata
+exiftool -Orientation#=3 four-colors.jpg -o .\four-colors-rotate-180-cw.jpg
+
+// Generate jpg picture with 270 cw rotation metadata
+exiftool -Orientation#=8 four-colors.jpg -o .\four-colors-rotate-270-cw.jpg
+
+```
```

Comment:
To simplify the tests I think it would be preferable for all four images (and perhaps the png) to have the same arrangement of colors, _post-rotation_. That way the expected colors will be the same for all four.

You could for example use imagemagick to generate four hard-rotated jpgs from `four-colors.png`, then use exiftool to soft-rotate them back.

Reply:
Done.